plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergDynamicPartitionPruningTest.java

```diff
@@ -66,9 +60,30 @@ protected void createPartitionedTable(String tableName, List<String> columns, Li
         getQueryRunner().execute(sql);
     }
 
+    @Override
+    protected void createPartitionedAndBucketedTable()
+    {
+        createPartitionedAndBucketedTable("t0_bucketed", ImmutableList.of("v0 decimal(2, 1)", "k0 integer"), ImmutableList.of("k0"), ImmutableList.of("v0"));
```

Comment:
Can we leave a code comment why overriding is required? e.g. 
```
// Use decimal type because Iceberg doesn't allow float type on bucket columns
```